src/routes/+page.svelte

```diff
@@ -1,15 +1,23 @@
 <script lang="ts">
+	import { onMount } from 'svelte';
 	import { Editor } from '$lib/index.js';
+	import type { EditorType } from '$lib/index.js';
 	import Nav from './nav.svelte';
-	
 
 	let saveStatus = 'Saved';
+	let editor: EditorType;
+
+	onMount(() => {
+		// Use the tiptap editor instance
+		console.log({ editor });
+	});
```

Comment:
We don't want the actual docs to have a console.log

Reply:
Done